app/code/Magento/ConfigurableProduct/Ui/DataProvider/Product/Form/Modifier/ConfigurablePanel.php

```diff
@@ -24,6 +24,7 @@ class ConfigurablePanel extends AbstractModifier
     const ASSOCIATED_PRODUCT_MODAL = 'configurable_associated_product_modal';
     const ASSOCIATED_PRODUCT_LISTING = 'configurable_associated_product_listing';
     const CONFIGURABLE_MATRIX = 'configurable-matrix';
+    const SHOW_PRODUCT_LISTING = 'show_product_listing';
```

Comment:
Add new constants as private ones. Please check the [Contributors Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/) for more details.
```suggestion
    private const SHOW_PRODUCT_LISTING = 'show_product_listing';
```

Reply:
fixed and commited